pkg/context/machine_context.go

```diff
@@ -62,7 +62,7 @@ func (c *VIMMachineContext) Patch(ctx context.Context) error {
 	return c.PatchHelper.Patch(ctx, c.VSphereMachine, patch.WithOwnedV1Beta2Conditions{Conditions: []string{
 		infrav1.VSphereMachineReadyV1Beta2Condition,
 		infrav1.VSphereMachineVirtualMachineProvisionedV1Beta2Condition,
-		clusterv1.PausedV1Beta2Condition,
+		clusterv1.PausedCondition,
```

Comment:
should this use clusterv1beta1.PausedV1Beta2Condition like the  other machine context?